Makefile

```diff
@@ -43,3 +49,21 @@ test-functional:
 freeze:
 	$(call print_msg, Freezing... )
 	python setup.py build
+
+update-flute-with-version:
+	sed -i 's/<auto-generated-rpm-version>/$(RPM_VERSION)/g' .flute
+
+rpmbuild/RPMS/x86_64/$(NAME)-$(RPM_VERSION)-$(RELEASE).x86_64.rpm: update-flute-with-version
+	flute --spec-only > $(NAME).spec
+	flute --spec $(NAME).spec
+	rm $(NAME).spec
+
+rpm: rpmbuild/RPMS/x86_64/$(NAME)-$(RPM_VERSION)-$(RELEASE).x86_64.rpm
+
+release: rpm
+	REPO_NAME="productivity" \
+	REPO_PATH="com/box" \
+	SERVICE_KIND="$(NAME)" \
+	SERVICE_VERSION="$(RPM_VERSION)-$(RELEASE)" \
+	rpm-to-artifactory rpmbuild/RPMS/x86_64/$(NAME)-$(RPM_VERSION)-$(RELEASE).x86_64.rpm
```

Comment:
I don't think this stuff should go in the main open source repo. We should probably have a private supporting repo with the Box-specific scripts. That probably applies to the .flute file as well since Flute is not (yet) open source.